lib/revision-selector.jsx

```diff
@@ -1,56 +1,89 @@
-import React from 'react'
+import React, { PropTypes } from 'react'
 import moment from 'moment'
+import { property } from 'lodash';
 
-export default React.createClass( {
+const lastUpdateOf = property( 'data.modificationDate' );
+const revisionSorter = ( a, b ) => lastUpdateOf( a ) - lastUpdateOf( b );
 
-	getDefaultProps: function() {
+const sortedRevisions = revisions => (
+	revisions
```

Comment:
this could simply be `orderBy( revisions, 'data.modificationDate', 'desc' )` (orderBy comes from lodash)